code/modules/assembly/infrared.dm

```diff
@@ -105,7 +116,7 @@
 
 /obj/item/device/assembly/infra/Topic(href, href_list)
 	..()
-	if(!usr.canmove || usr.stat || usr.restrained() || !in_range(loc, usr))
+	if(usr.incapacitated() || !in_range(loc, usr))

```

Comment:
this simplify the code a bit and fixes not being able to use the device while buckled.